actors/abi/epoch.go

```diff
@@ -0,0 +1,8 @@
+package abi
```

Comment:
Thanks for extracting these. I don't think `abi` is the right package for the constants (it's for types). Please move to `builtin/network.go`